doc/architecture/decisions/0008-lookups.md

```diff
@@ -0,0 +1,46 @@
+# 8. Lookups
+
+## Context
+
+We need to be able to specify a lookup component within a runner schema. These are to be used to provide a dropdown list of suggestions and simple entry before a user enters the full answer text.
+
+## Decision
+
+We will define a new answer type 'Lookup' within the schema which identifies the answer should be rendered as a lookup component.
+
+    {
+        id: 'religion-question',
+        title: 'What is your religion?',
+        description: 'This question is voluntary',
+        type: 'General',
+        answers: [
+          {
+            id: 'religion-answer',
+            mandatory: false,
+            label: 'Religion',
+            type: 'Lookup',
+            query_path: 'path/to/query'
+          }
+        ]
+    }
+
+The answer behaves as a Textfield when JavaScript is disabled and specifies a query_path for the frontend component to retrieve suggestions from when enabled.
```

Comment:
I think it behaves as a Textfield regardless of Javascript. The Javascript will add the suggestions list below the textfield.

![image](https://user-images.githubusercontent.com/1754822/57444375-ef077a80-7247-11e9-8188-3d5afaad8cba.png)